src/game/savegame/savegame_bson.c

```diff
@@ -1267,6 +1300,11 @@ bool Savegame_BSON_LoadFromFile(MYFILE *fp, GAME_INFO *game_info)
         goto cleanup;
     }
 
+    if (!SaveGame_BSON_LoadMusicTrackFlags(
+            json_object_get_array(root_obj, "music_track_flags"))) {
+        goto cleanup;
+    }
+
```

Comment:
Since we used version bump in the previous PR we should also follow the same process here. I'd reuse version 3 and not bump the version to 4, because its introduction hasn't been released yet.

Reply:
We ended up not bumping the savegame version.

Reply:
To close this loop, I added back in the savegame version bump since we decided to do it.